src/imagePickerIOS.rei

```diff
@@ -0,0 +1,15 @@
+let canUseCamera: (unit => unit) => unit;
+
+let canRecordVideos: (unit => unit) => unit;
+
+[@bs.deriving abstract]
+type error = {
```

Comment:
For the rei, would it be possible to expose only the abstract `type error` and the getters?